### ISSUE TITLE
Add selectable R/G/B channel for BW image

### ISSUE THREAD

**Issue by zaskokus** [jomjol/AI-on-the-edge-device#3183]
### The Feature

As the title says. For example my meter has only black and red digits on white background. Being able to use only red channel for BW image would greatly increase contrast of the picture, thus recognition quality - this is especially an issue for my analog wheel-arrow thingie.

**Comment by caco3** [comment]
Did you try if you could use [Gray Scale](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-grayscale) mode to work around this?

**Comment by zaskokus** [comment]
Yes, I am using gray scale already, and that's what I'm trying to suggest as an extension of it - a selectable channel(s) for gray scale.

**Comment by caco3** [comment]
Not sure what [CamSpecialEffect](https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#parameter-camspecialeffect) actually does, but maybe give it a try.

@SybexX FYI

**Comment by zaskokus** [comment]
Doesn't look like it changed anything at all. For reference I tied the same with an image editor and my red digits were much darker when using red channel for BW than all RGB for BW. The result is near identical to RGB based BW.

**Comment by SybexX** [comment]
The special effects are handled by the OV2640 itself.
I'm assuming that you have to retrain the models for the special effects.
